### PR TITLE
Autoscroll tablature during while playing

### PR DESCRIPTION
**Pull request by agourlay**
Tackling https://github.com/agourlay/ruxguitar/issues/7

The implementation is slightly hacky but seems to get the job done.

Looking forward to get feedback on the resulting scroll :)

### CODE REVIEW

**Comment by agourlay** [comment]
@VladasZ Could you maybe give this branch a try tell me how it goes for you :)

File: src/ui/canvas_measure.rs [hunk 1]
```diff
@@ -65,14 +70,13 @@ impl CanvasMeasure {
...
+            vertical_measure_height,
         }
     }
 
     pub fn view(&self) -> Element<Message> {
-        let string_count = self.song.tracks[self.track_id].strings.len();
-        let vertical_measure_height = STRING_LINE_HEIGHT * (string_count - 1) as f32;
         let canvas = Canvas::new(self)
-            .height(vertical_measure_height + FIRST_STRING_Y * 2.0)
+            .height(self.vertical_measure_height)
```
Comment (agourlay) [inline src/ui/canvas_measure.rs:79]: avoid recomputing the constant for each redraw

**Comment by VladasZ** [comment]
Hi! I tried it and the scroll does work. A small thing that bothers me is that it scrolls after each line. In comparison guitar pro scrolls only at the end of the page and in my opinion it is easier to read with such scroll. But still this implementation is much better than no scroll at all and it should be merged and maybe improved later.

Also I found a bug while testing: https://github.com/agourlay/ruxguitar/issues/11

**Comment by agourlay** [comment]
Thanks a lot for the feedback!

I was able to fix #11

>  But still this implementation is much better than no scroll at all and it should be merged and maybe improved later.

Also agree with you there :+1:

**Pull request merged**